data_steward/validation/main.py

```diff
@@ -1045,15 +1044,11 @@ def run_retraction_cron():
     dataset_ids = bq_utils.get_retraction_dataset_ids()
     logging.info(f"Dataset id/s to target from env variable: {dataset_ids}")
     logging.info(f"Running retraction on BQ datasets")
-    if output_project_id:
-        # retract from output dataset
-        retract_data_bq.run_bq_retraction(output_project_id, sandbox_dataset_id,
-                                          project_id, pid_table_id, hpo_id,
-                                          dataset_ids, retraction_type)
+
```

Comment:
Thanks, @lrwb-aou. I will update `run_bq_retraction()` for that because the refactored `run_bq_retraction()` takes only one BQ project as its argument. It currently assumes the lookup table and the datasets for retraction are in the same project.

Reply:
I suggest not making changes yet @hiro-mishima .  We're still trying to determine who is responsible for the datasets in that project and what the workflow should be.